.github/assets/setup.sh

```
@@ -0,0 +1,16 @@
+export SB="/tmp/zerostarter"
+rm -rf "$SB"
+mkdir -p "$SB/acme"
+
+ZS_BIN="$PWD/packages/cli/dist/bin/index.mjs"
+zerostarter() { node "$ZS_BIN" "$@"; }
+
+cd "$SB/acme"
+autoload -Uz add-zsh-hook
+# blank line before the prompt; %1~ shows just the project dir (acme), not the /tmp path; prompt renders cyan, reset to default before each command's output
+PROMPT=$'\n%F{cyan}%1~ ❯ '
+__zs_demo_reset() { print -n $'\e[0m' }
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this function.

(SC1009)